client/src/views/SignupNonProfit.tsx

```diff
@@ -1,7 +1,836 @@
+import SvgSignUpFinishedStep from '../components/Icons/SvgSignUpFinishedStep';
+import {
+  Box,
+  Select,
+  MenuItem,
+  Avatar,
+  TextField,
+  OutlinedInput,
+  SelectChangeEvent,
+  Chip,
+} from '@mui/material';
+
+interface UserSignupData {
+  organization_name: string;
+  organization_phone: string;
+  street: string;
+  city: string;
+  state: string;
+  zip_code: string;
+  employer_identification_number: string;
+  irs_classification: string;
+  first_name: string;
+  last_name: string;
+  role: string;
+  email: string;
+  password: string;
+  accept_terms?: boolean;
+  email_notification_opt_out?: boolean;
+  website: string;
+  instagram: string;
+  facebook: string;
+  twitter: string;
+  focusAreas: string[];
+  bio: string;
+}
+
+const initialFormData: UserSignupData = {
+  organization_name: '',
+  organization_phone: '',
+  street: '',
+  city: '',
+  state: '',
+  zip_code: '',
+  employer_identification_number: '',
+  irs_classification: '',
+  first_name: '',
+  last_name: '',
+  role: '',
+  email: '',
+  password: '',
+  accept_terms: false,
+  email_notification_opt_out: false,
+  website: '',
+  instagram: '',
+  facebook: '',
+  twitter: '',
+  focusAreas: [],
+  bio: '',
+};
+
+const useStyles = makeStyles()((theme: Theme) => ({
+  sideImg: {
+    backgroundImage: `url("${placeholderImg}")`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center center',
+    backgroundRepeat: 'no-repeat',
+    borderTopRightRadius: '15px',
+    borderBottomRightRadius: '15px',
+  },
+  signUpContainer: {
+    margin: theme.spacing(5),
+  },
+  button: {
+    borderRadius: 0,
+    height: 44,
+    textTransform: 'none',
+    backgroundColor: '#C4C4C4',
+    color: 'white',
+  },
+  header: {
+    fontWeight: 'bold',
+    paddingBottom: '40px',
+  },
+  input: {
+    height: 44,
+    border: '1px solid #C4C4C4',
+    borderRadius: 10,
+    boxSizing: 'border-box',
+    padding: theme.spacing(1),
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+    fontSize: 14,
+    marginBottom: 20,
+  },
+  label: {
+    color: '#000000',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+  chip: {
+    boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.25)',
+    height: 44,
+  },
+}));
 
 function SignupNonProfit() {
-  return <SignUpUserAndNonprofit />;
+  const { classes } = useStyles();
+  const [selectedChips, setSelectedChips] = React.useState<string[]>([]);
+  const [disableNext, setDisableNext] = React.useState<boolean>(true);
+  const [activeStep, setActiveStep] = React.useState<number>(0);
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [emailError, setEmailError] = React.useState<string>('');
+  const [formData, setFormData] = React.useState(initialFormData);
+  const { user, setUser } = React.useContext(UserContext);
+  const modalContext = useContext(ModalContext);
+  const { openModal } = modalContext;
+  const { control } = useForm();
+  const steps = [
```

Comment:
This can probably be placed outside the function, since this doesn't change.

Reply:
This component is pretty big.  It might be good idea to refactor this into multiple smaller components and this can be a parent or container component.